### ISSUE TITLE
Hamburger Navigation Menu is Broken, Again

### ISSUE THREAD

**Issue by dzoladz** [alaoweb/2020.alaoweb.org#186]
Major issue for mobile clients.
<img width="619" alt="Screen Shot 2020-08-31 at 10 44 52 AM" src="https://user-images.githubusercontent.com/10561752/91754162-f4a07680-eb96-11ea-9455-bb8db39b9703.png">

**Comment by kenirwin** [comment]
When I build the current site on localhost, I get the hamburger menu. But it's not building on Netlify. 

@dzoladz - any idea about where I should poke at this to get a different answer?

**Comment by kenirwin** [comment]
I'm thinking maybe I submit some PRs reverting the site to old versions; not to actually revert the master, but to check out the Netlify previews to see where things went wrong. Doing so on localhost isn't working out.

**Comment by dzoladz** [comment]
The issue was in one of the Netlify plugins that I had enabled for the builds from the `master` branch. There were several reasons that I had enabled the some basic checking on the Netlify end, but it's not living up to its promise. I've also disabled the ImageOptum plugin from the build pipeline, in favor of using what Netlify calls _Asset Optimizations_:

Here are the Asset Optimizations enabled for production builds:
- **Pretty URLs:** Rewrite link URLs to pretty URLs (/about.html -> /about, /about/index.html -> /about/)
- **Bundle CSS:** Concatenate consecutive CSS files together to reduce HTTP requests.
- **Minify CSS:** Run CSS through a minifier to reduce file size.
- **Bundle JS:** Concatenate consecutive JS files together to reduce HTTP requests.
- **Minify JS:** Run JS through a minifier to reduce file size.
- **Compress Images:** Run all images through lossless image compression.

Since Asset Optimizations are maintained by Netlify - rather than a third-party, as plugins are - I'm confident that we'll see fewer issues going forward.

I've cleaned out our Slack channel and GitHub, so we should be all set to go.
